README.md

```diff
@@ -30,13 +30,13 @@ cases.
 
 ## Prerequisites
 
-| Name | Requirement |
-| :--- | :------ |
+| Name                                                           | Requirement          |
```

Comment:
Android studio Dolphin patch 1 contains nice plugin to format MD files :)

Reply:
ok :)